MODULE.bazel

```diff
@@ -0,0 +1,16 @@
+module(
+    name = "com_google_googleapis",
+    version = "0.0.0",
+)
+
+bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+
+switched_rules = use_extension("//:extensions.bzl", "switched_rules")
+switched_rules.use_languages(
+    cc = True,
+    go = True,
+    java = True,
+    python = True,
```

Comment:
```suggestion
    python = True,
    grpc = True,
```

It looks to me if you add this some other deps are missing

Reply:
maybe blocked on https://github.com/bazelbuild/bazel-central-registry/issues/353